### PR TITLE
fix(DB/GOSSIP) Duplicate gossip option

### PR DESCRIPTION
**Pull request by tkn963**
## Changes Proposed:
-  Remove duplicate gossip option for Sergeant Bly
-  Remove unused broadcast_text for said gossip option

## Issues Addressed:

- 

## SOURCE:

- 

## Tests Performed:

- 

## How to Test the Changes:

1. Initiate the stair event in Zul'Farrak
2. Sergeant Bly no longer has a gossip option while at the top of the stairs
3. Sergeant Bly now only has one option that is used to insult him

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by tkn963** [comment]
My second commit is a removal of the locale for the gossip option. I'm saying this because this is my first PR ever and I forgot to write different information for the commit.

File: data/sql/updates/pending_db_world/rev_1621437037316104400.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621437037316104400');
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID`=941 AND `OptionID`=1;
+DELETE FROM `gossip_menu_option_locale` WHERE `MenuID`=941 AND `OptionID`=1;
+DELETE FROM `broadcast_text` WHERE `ID`=4165;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1621437037316104400.sql:5]: Dont delete broadcast_text ever

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1621437037316104400.sql (hunk 1 above)
Comment (tkn963) [reply data/sql/updates/pending_db_world/rev_1621437037316104400.sql]: My apologies, I changed it and now I know for the future

**Review by Kitzunu — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@tkn963 can you please specify what tests have you performed with this PR. Have you tested this in game?

> Tests Performed:

**Comment by tkn963** [comment]
> 
> 
> @tkn963 can you please specify what tests have you performed with this PR. Have you tested this in game?
> 
> > Tests Performed:

Yes I have tested the event in question a couple of times to make sure they work as they should. I am sorry for forgetting to mention this above. Are these issues that I should look into that I haven't noticed?

**Comment by UltraNix** [comment]
Superseded by #5952

**Pull request closed without merging**